smack-core/src/main/java/org/jivesoftware/smack/ConnectionConfiguration.java

```diff
@@ -577,7 +578,7 @@ public boolean isEnabledSaslMechanism(String saslMechanism) {
         private CharSequence username;
         private String password;
         private Resourcepart resource;
-        private String language = "en";
+        private Locale language = Locale.ENGLISH;
```

Comment:
Is there any reason to not use `Locale.getDefault()` to initialize the value?

Reply:
I didn't dare to change the default, which was "en". If you feel that using the default locale is good, and it's not going to break anything, I can patch it in.

Reply:
I'm pretty sure, that there may be *some* tests that might break on non-english systems.
There was one such case pretty recently, but fixing those tests in the first place is probably a good idea :D

Reply:
What @vanitasvitae said :)

Reply:
The "nice" thing about that is that now we are leaking the environment of the user into the test setup, and from my experience, it will only break on locales that are not used by the maintainers in the first place, like Turkish (capital "i"), or Arabic (number parsing / formatting will bite you!). Feel free to rework the test setup to iterate through all existing locales, but I'm keeping this status-quo as is in this PR.